dbos/system_database.py

```diff
@@ -4,17 +4,7 @@
 import threading
 import time
 from enum import Enum
-from typing import (
-    TYPE_CHECKING,
-    Any,
-    Dict,
-    List,
-    Literal,
-    Optional,
-    Sequence,
-    TypedDict,
-    cast,
-)
+from typing import Any, Dict, List, Literal, Optional, Sequence, TypedDict, cast
```

Comment:
The reformatter is obnoxious, very difficult to see what changed and likely to induce merge conflicts.

Reply:
It's also not very consistent across runs..

Reply:
I wish it had hysteresis, like if more than 5 split it, if only one, condense it, otherwise leave it the way it is.  So it wouldn't flap back and forth.